mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -519,4 +520,18 @@ def test_automatic_vmrk_sfreq_recovery():
                        read_annotations(vmrk_path, sfreq=1000.0))
 
 
+@testing.requires_testing_data
+def test_event_id_stability_when_save_and_fif_reload(tmpdir):
+    """Test load events from brainvision annotations when read_raw_fif."""
+    fname = str(Path(tmpdir, 'bv-raw.fif'))
```

Comment:
why `Path` and not just `fname = op.join(str(tmpdir), 'bv-raw.fif')` like we do everywhere else?

Reply:
yes. I should do it for consistency here. I was tired of `str(tmpdir)` and I thought that using Path would solve it. But the problem is with the reader rather than with `tmpdir`. 

I think I'll open a PR to add Path to the readers. Path is nice.
https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/

But here I just forgot to revert.